odoo/tests/test-uninstalls.py

```diff
@@ -56,6 +57,10 @@ def parse_args():
         help="Comma-separated list of paths to directories containing extra Odoo modules")
     parser.add_argument("--uninstall", "-U", type=str,
         help="Comma-separated list of modules to uninstall/reinstall")
+    parser.add_argument("--test-scripts", type=str,
```

Comment:
The command is already called `test_uninstall`, I suggest alternative option names like `test` or `script` or `standalone`.